signbank/video/tests/test_views.py

```diff
@@ -196,3 +197,116 @@ def test_upload_glossvideo_with_custom_redirect(self):
         response = self.client.post(reverse('video:upload_glossvideo_gloss'), {
             'videofile': self.testfile, 'gloss': self.testgloss.pk, 'video_type': self.video_type.machine_value, 'redirect': reverse('video:manage_videos')})
         self.assertEqual(response.url, reverse('video:manage_videos'))
+
+class ExportGlossvideoCsvTestCase(TestCase):
+    def setUp(self):
+        # Create user and add permissions
+        self.user = User.objects.create_user(
+            username="test", email=None, password="test")
+        permission = Permission.objects.get(codename='export_csv')
+        self.user.user_permissions.add(permission)
+        self.user.save()
+        # Create client with export permission.
+        self.client = Client()
+        self.client.login(username="test", password="test")
```

Comment:
**suggestion**: because we're not testing the password backend works here, we can instead call [force_login](https://docs.djangoproject.com/en/4.2/topics/testing/tools/#django.test.Client.force_login) and provide the user instance